pkg/build/controller/controller.go

```diff
@@ -208,7 +208,7 @@ func (bc *BuildController) resolveOutputDockerImageReference(build *buildapi.Bui
 		stream, err := bc.ImageStreamClient.GetImageStream(namespace, streamName)
 		if err != nil {
 			if errors.IsNotFound(err) {
-				return "", fmt.Errorf("the referenced output image stream %s/%s does not exist", namespace, streamName)
+				return "", fmt.Errorf("the referenced output image stream %s/%s does not exist.", namespace, streamName)
```

Comment:
what @deads2k said.  also i don't have enough hoops in my life, so i figured i'd find some more to jump through.